test/extended/openstack/cpms.go

```diff
@@ -109,17 +126,30 @@ var _ = g.Describe("[sig-installer][Suite:openshift/openstack] ControlPlane Mach
 				subnetName := subnet.Get("filter.name").String()
 				machineSubnets[subnetName] = struct{}{}
 			}
-			if machineNovaAvailabilityZone != "" {
-				machineNovaAvailabilityZones = append(machineNovaAvailabilityZones, machineNovaAvailabilityZone)
+			if machineNovaAz != "" || machineCinderAz != "" || machineVolumeType != "" {
+				machinesFDs = append(machinesFDs, FailureDomain{
+					ComputeZone: machineNovaAz,
+					VolumeZone:  machineCinderAz,
+					VolumeType:  machineVolumeType,
+				})
 			}
+
+			for _, network := range objects(machine.Get("spec.providerSpec.value.networks")) {
+				subnet := network.Get("subnets").String()
+
+				if subnet == "" {
+					machineNetworks = append(machineNetworks, network["uuid"].(string))
+				}
```

Comment:
You don't use that `subnet` variable, am I right?

```suggestion
				if network.Get("subnets").String() == "" {
					machineNetworks = append(machineNetworks, network.Get("uuid").String())
				}
```

Reply:
right. Updated